dev/cli/push-build.ts

```diff
@@ -2,11 +2,33 @@ import { shSync as sh } from "../sh";
 
 const args = process.argv.slice(2);
 
-if (args.length < 2) {
-	process.stderr.write(`Usage ${process.argv[1]} [app-name] [...paths]`);
+if (args.length < 1) {
+	process.stderr.write(
+		`Usage ${process.argv[1]} [--deploy-staging] [--origin GIT-ORIGIN] [...paths]`
+	);
 	process.exit(1);
 }
 
+const commitPaths = [];
+let deployStaging = false;
+let origin = "origin";
+
+for (let i = 0; i < args.length; i++) {
+	const arg = args[i];
+	if (arg === "-ds" || arg === "--deploy-staging") {
```

Comment:
AFAIK, the short variant of a flag is typically a single letter. What do you think about changing the flag to `--deploy-staging` (or just `--staging`) and `-s`?